tests/acceptance/tests/Search/ProductSearch.spec.ts

```diff
@@ -9,51 +9,65 @@ test('Customer is able to search products in shop', { tag: '@Search' }, async ({
     IdProvider,
     InstanceMeta,
 }) => {
-        const productNameSuffix1 = IdProvider.getIdPair().uuid;
-        await TestDataService.createBasicProduct({
-            name: `Bottle${productNameSuffix1}`,
-        });
-        await TestDataService.createBasicProduct({
-            name: `Bowl${productNameSuffix1}`,
-        });
+    const productNameSuffix1 = IdProvider.getIdPair().uuid;
+    await TestDataService.createBasicProduct({
+        name: `Bottle${productNameSuffix1}`,
+    });
+    await TestDataService.createBasicProduct({
+        name: `Bowl${productNameSuffix1}`,
+    });
 
-        await test.step('Customer searches with an invalid input and sees no results', async () => {
-            await ShopCustomer.goesTo(StorefrontHome.url());
-            await ShopCustomer.attemptsTo(SearchForTerm('thisShouldNotFindAnything'));
-            await ShopCustomer.expects(StorefrontSearchSuggest.searchSuggestNoResult).toBeVisible();
+    await TestDataService.clearCaches();
```

Comment:
This is actually important for the search, it does not seem to be necessary for the listing though